pyproject.toml

```diff
@@ -36,9 +36,12 @@ dependencies = [
   "webencodings"
 ]
 
+[project.scripts]
```

Comment:
note: pip install will use the `pyproject.toml` *instead of* `setup.py` (they are equivalent-ish things).

it's misleading to have both, we should just use one or the other.

for now, this just patches some critical fields so we can generate the executable and fill out equivalent metadata. 

This should fix the `hatch run install` flow.